### ISSUE TITLE
Too slow fo webui

### ISSUE THREAD

**Issue by massyzs** [buaacyw/GaussianEditor#67]
Why doing operation based on webui is so slow? I run it on L40s server which should be pretty fast for gaussian splatting task. Rotate the angle now takes minutes

**Comment by hashirupramuditha** [comment]
Yeah exactly, is there any way to make the editing operations, at least the rotating of the scene faster?
